gdp-src-build/conf/templates/dragonboard-410c.local.conf

```diff
@@ -0,0 +1,5 @@
+include templates/local.inc
+
+MACHINE = "dragonboard-410c"
+
+ACCEPT_EULA_dragonboard-410c = "1"
```

Comment:
The general policy we are following is to make GDP easy to consume. We can inform users about this EULA in the target board wiki page.